src/BitOasis/Common/CurrencyPair/CurrencyCodePair.php

```diff
@@ -880,8 +970,8 @@ class CurrencyCodePair implements Pair {
 		'TRX-USDT', 'TRX-BTC', 'TRX-ETH',
 
 		// TUSD is De-listed (keeping for historical support)
-		'TUSD-AED', 'TUSD-SAR', 'TUSD-TRY', 'TUSD-USD',
-		'TUSD-USDT',
+//		'TUSD-AED', 'TUSD-SAR', 'TUSD-TRY', 'TUSD-USD',
+//		'TUSD-USDT',
```

Comment:
remove comment, just keep `TUSD-TRY` for historical support

Reply:
Done